specification/repository.md

```diff
@@ -104,8 +104,8 @@ approval is granted, GDI projects MUST NOT cut a GA release.
   - MUST have getting started information in `README.md`
   - MUST have troubleshooting information in `README.md`
   - MUST have license information in `README.md`
-- MUST have a `RELEASING.md` documenting the release process
-  - SHOULD be able to release by pushing a tag
+- MUST NOT have a `RELEASING.md` file or any release process documentation
```

Comment:
I think that Go libraries could be an exception here.
@owais @MrAlias do you agree?

Reply:
I think we should not prevent from having a RELEASING.md file. We should restrict projects from documenting internal/private release processes instead. Projects can have a RELEASING.md file and document how to do it publicly when applicable (e.g, Go libs), have it link to internal docs or simply document that release jobs are private.